spec/transaction/tags.yaml

```diff
@@ -9,13 +9,108 @@ tags:
+      | application/x.csv+zip | Zipped CSV | .zip |
+      | text/csv | CSV | .csv |
+      | application/geopackage+sqlite3 | Geopackage | .gpkg |
+      | application/x.geopackage+zip | Zipped Geopackage | .zip |
+      | application/geo+json | GeoJSON (<span style="color: red">does not support auto HTTP compression !</span>) | .json, .geojson |
+      | application/x.geojson+zip | Zipped GeoJSON | .zip |
+      | application/x.interlis; version=2.3 | Interlis 2 | .xtf, .xml |
+      | application/x.interlis+zip; version=2.3 | Zipped XTF (2.3) | .zip |
+      | application/x.interlis; version=1 | Interlis 1 | .itf |
+      | application/x.interlis+zip; version=1 | Zipped ITF | .zip |
+      | image/tiff; application=geotiff; profile=cloud-optimized | Cloud Optimized GeoTIFF (COG) | .tiff, .tif |`
+      | application/pdf | PDF | .pdf |
+      | application/x.pdf+zip | Zipped PDF | .zip |
+      | application/json | JSON | .json |
+      | application/x.json+zip | Zipped JSON | .zip |
+      | application/x-netcdf | NetCDF | .nc |
+      | application/x.netcdf+zip | Zipped NetCDF | .zip |
+      | application/xml | XML | .xml |
+      | application/x.xml+zip | Zipped XML | .zip |
+      | application/vnd.mapbox-vector-tile | mbtiles | .mbtiles |
+      | text/plain | Text | .txt |
+      | text/x.plain+zip | Zipped text | .zip |
+      | application/x.dwg+zip | Zipped DWG | .zip |
+      | application/zip | Generic Zip File | .zip |
+      | image/tiff | TIFF | .tiff, .tif |
+      | image/jpeg | JPEG | .jpeg, .jpg |
+      | image/png | PNG | .png |
+      | application/vnd.sqlite3 | sqlite | .sqlite |
+
   - name: Asset Upload Management
     description: |
-      Asset file can be uploaded via the STAC API using the following requests.
+      Asset files are uploaded via the STAC API using the API requests described in this chapter.
+
+      - [Authentication](#section/Authentication)
+      - [Compression](#section/Compression)
+      - [Example](#section/Example)
+
+      <span style="color:red">***IMPORTANT NOTES:***</span>
+        - <span style="color:red">*Text based files bigger than 10 MB should use HTTP compression,
+          see [HTTP Compression](#section/HTTP-Compression)*</span>
+
+
+      ## Authentication
+
+      POST/PUT requests require authentication as described in [here](#tag/Authentication).
+
+      ## HTTP Compression
+
+      Text based files between *1 MB* and *10 MB* are automatically compressed during download using *gzip* or *br* based on the
+      `Accept-Encoding` header of the request. But note that this compression is only done for standard Media Type (see
+      [File types that CloudFront compresses](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/ServingCompressedFiles.html#compressed-content-cloudfront-file-types)).
+
+      <span style="color: red">It is highly recommended to not upload text based files bigger than *10 MB* !</span>
```

Comment:
I'd probably formulate this a bit different and less 'dramatic': the use case of having big files on stac is in general not uncommon, so we can not say it's highly unrecommended. 
The general recommendation for files bigger than 10Mb, regardless of the type, should be to use a compressed media type. If this is not possible (as e.g. for jsons directly used in a browser app), the file should be compressed upfront or split.